### ISSUE TITLE
Add pathways and key events to datanodes.tsv

### ISSUE THREAD

**Issue by AlexanderPico** [wikipathways/GPML-Repo#127]
The participants table is empty for WP5227 because the datanodes.tsv is empty... because all the datanodes are either pathways or "key events".

https://classic.wikipathways.org/index.php/Pathway:WP5227

**Comment by AlexanderPico** [comment]
@egonw @mkutmon @hasanbalci I think this is a meta-data-action issue, but I'm not sure.  Ideas on how to get these other datanode types into the TSV?  Any arguments against including these?

**Comment by egonw** [comment]
Yeah, maybe a good moment to pick this up. We're going to broaden that list for GPML 2021 anyway.
